src/test/java/io/logz/jmx2graphite/GraphiteClientTest.java

```diff
@@ -67,7 +67,7 @@ public void testOnServerRestart() throws InterruptedException {
         int connectTimeout = 1000;
         int socketTimeout = 1000;
         GraphiteClient client = new GraphiteClient("bla-host.com", "bla-service", "localhost",
-                                                   port, connectTimeout, socketTimeout, 20000,
+                                                   port, false, connectTimeout, socketTimeout, 20000,
```

Comment:
Can you please add an integration test to see that the functionality works?

Reply:
Ah, sorry, setting up an ssl-enabled graphite is fairly complicated, and doing it in an integration test more so, and I don't have the time right now to get it working. I understand if you don't want to merge without a test though. Thanks!

Reply:
@elahrvivaz Even if you use testcontainers? If you already did once for manual testing, it's only a matter of writing the configuration file for Graphite no?

Reply:
ah, well I didn't set up the instance I've been testing on, and I'm not familiar with testcontainers :(

Reply:
@elahrvivaz Testcontainers is fairly easy to work with. You have a great example at GraphiteITest. Take a few minutes to review it - I'll happy to assist with any questions you may have.